app/src/androidTest/java/com/okta/idx/android/SelfServiceRegistrationTestEndToEnd.kt

```diff
@@ -50,18 +51,26 @@ import org.junit.runner.RunWith
 class SelfServiceRegistrationTestEndToEnd {
     @get:Rule val activityRule = ActivityScenarioRule(MainActivity::class.java)
 
+    var profile: A18NProfile? = null
+
+    @After
+    fun deleteProfile() {
+        profile?.let {
+            A18NWrapper.deleteProfile(it)
+        }
+    }
+
     // Mary signs up for an account with Password, sets up required Email factor, then skips
     // optional SMS.
     @Test fun scenario_4_1_1() {
-        val profile = A18NWrapper.createProfile()
-
+        profile = A18NWrapper.createProfile()
```

Comment:
Can we do this in a `@Before`? The other tests will need this too.

Reply:
4.1.3 does not use a18n profiles

Reply:
Should we still put this into `@Before`

Reply:
Leaving it as is works given that information.